### ISSUE TITLE
lov suggest check fails

### ISSUE THREAD

**Issue by seebi** [vocol/scor#26]
http://lov.okfn.org/dataset/lov/suggest?q=http%3A%2F%2Fpurl.org%2Feis%2Fvocab%2Fscor%23

we should host it somewhere else than in github ...

**Comment by np00** [comment]
Yes, it is already on my todo list for some time :) It is of low priority to me, because I don't see yet others using/references our ontology (yet).
